jwst/extract_2d/nirspec.py

```diff
@@ -173,26 +188,18 @@ def process_slit(input_model, slit, exp_type):
         Input data model. The ``CubeModel`` is used only for TSO data, i.e.
         ``NRS_BRIGHTOBJ`` exposure or internal lamp exposures with lamp_mode
         set to ``BRIGHTOBJ``.
-    slit_name : str or int
```

Comment:
doesn't parameter `slit` need to be here?

Reply:
Yes, not sure why ruff didn't catch that